src/test/java/com/epam/tc/hw3/BaseTest.java

```diff
@@ -0,0 +1,32 @@
+package com.epam.tc.hw3;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+
+public class BaseTest {
+
+    protected WebDriver driver;
+
+    @BeforeClass()
+    public void oneTimeSetUp() {
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
```

Comment:
Instance of the driver should be created before each test

Reply:
Instance of the driver is created before each test